src/guides/v2.4/graphql/mutations/add-gift-registry-registrants.md

```diff
@@ -0,0 +1,109 @@
+---
+group: graphql
+title: addGiftRegistryRegistrants mutation
+
+---
+The `addGiftRegistryRegistrants` mutation add a registrant to the specified gift registry.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+## Syntax
+
+```graphql
+mutation {
+  addGiftRegistryRegistrants(
+    giftRegistryUid: ID!,
+    registrants: [AddGiftRegistryRegistrantInput!]!
+  ) {
+    AddGiftRegistryRegistrantsOutput
+  }
+}
+```
+
+## Example usage
+
+The following example adds a registrant to the specified gift registry.
+
+**Request:**
+
+```graphql
+mutation {
+  addGiftRegistryRegistrants (
+    giftRegistryUid: "W9YcRai9JmzGglqP3p0USodTTM3BmjjY", 
+    registrants: {
+        firstname: "Monica"
```

Comment:
Would suggest using a name like "Jane Doe" as "Monica Resendez" is a real person.